### PR TITLE
Fan02 and Mysterium Clothing additions

### PR DESCRIPTION
**Pull request by Hazado**
Adds 28 shirts for Mysterium from 2000 - 2022
Modifies Mysterium 2021 short sleeve shirt to be included into new Mysterium category
Adds reward fleece for Eder Naybree
Adds default long and short sleeve shirt for Guild of Instructors

### CODE REVIEW

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 1]
```diff
@@ -0,0 +1,2358 @@
...
+            {
+              "name": "Arm",
+              "layers": {
+                "0": "mtorso_shortsleevearm_base*0#0.hsm",
+                "1": "mtorso_shortsleevearm_skin*0#0.hsm",
+                "8": "mtorso_shortsleevearm_tint1*0#0.hsm"
+              }
+            }
+          ]
+        },{
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:309]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 2]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:349]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 3]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:431]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 4]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:471]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 5]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:511]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 6]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:551]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 7]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:591]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 8]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:631]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 9]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:671]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 10]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:711]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 11]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:751]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 12]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:791]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 13]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:831]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 14]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:871]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 15]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:911]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 16]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:951]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 17]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:991]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 18]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1031]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 19]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1111]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 20]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1151]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 21]
```diff
@@ -0,0 +1,2358 @@
...
+              "layers": {
+                "0": "mtorso_shortsleevearm_base*0#0.hsm",
+                "1": "mtorso_shortsleevearm_skin*0#0.hsm",
+                "8": "mtorso_shortsleevearm_tint1*0#0.hsm"
+              }
+            }
+          ]
+        }
+      ]
+    },{
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1193]:
```suggestion
    },
    {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 22]
```diff
@@ -0,0 +1,2358 @@
...
+          "icon": "Icon_UTorso_Shirt_Mysterium.png",
+          "tint1": [0.2863, 0.2863, 0.2863],
+          "tint2": [1, 1, 1],
+          "desc": "GroupIcon; GroupName = Mysterium",
+          "text": "Mysterium T-Shirts",
+          "meshes": [
+          {},{},{}
+          ],
+          "elements": [
+          ]
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1298]:
```suggestion
          "meshes": [{},{},{}],
          "elements": []
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 23]
```diff
@@ -0,0 +1,2358 @@
...
+        },{
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1299]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 24]
```diff
@@ -0,0 +1,2358 @@
...
+            },
+            {
+              "name": "Arm",
+              "layers": {
+                "1": "mtorso_shortsleevearm_skin*0#0.hsm",
+                "8": "fsleeve04t02l1*0#0.hsm"
+              }
+            }
+          ]
+        },{
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1338]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 25]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1377]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 26]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1416]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 27]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1455]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 28]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1494]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 29]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1533]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 30]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1613]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 31]
```diff
@@ -0,0 +1,2358 @@
...
+          "name": "FReward_SSMysterium2006Australia",
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1574]: This name is too long to fit in an SDL variable. I thought that the Australia Mysterium was called "Mystralia" BTW.

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 32]
```diff
@@ -0,0 +1,2358 @@
...
+            },
+            {
+              "name": "Arm",
+              "layers": {
+                "1": "mtorso_shortsleevearm_skin*0#0.hsm",
+                "8": "fsleeve04t02l1*0#0.hsm"
+              }
+            }
+          ]
+        },{
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1652]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 33]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1691]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 34]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1730]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 35]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1769]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 36]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1808]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 37]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1847]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 38]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1925]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 39]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:1964]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 40]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:2003]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 41]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:2042]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 42]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:2081]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 43]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:2120]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 44]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:2198]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 45]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:2237]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 46]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:2276]:
```suggestion
        },
        {
```

File: sources/avatar/clothing_mysterium/mysterium.json [hunk 47]
```diff
@@ -0,0 +1,2358 @@
...
```
Comment (Hoikas) [inline sources/avatar/clothing_mysterium/mysterium.json:2315]:
```suggestion
        },
        {
```

**Comment by Hazado** [comment]
> I'm not sure how these shirts are planned to be granted, but the `MReward` and `FReward` prefixes indicate you plan to grant these via the vault. Unfortunately, there is a 31 character limit for the total amount of clothing you can grant at any one time, so this seems a little suspect.

Doobes plans on a pub area to get all the shirts.

**Comment by Hoikas** [comment]
Ok, so we probably need to drop the `[MF]Reward]` prefix in favor of matching... whatever the non-vault reward clothing does.

File: sources/avatar/clothing_fan02/Fan02.json [hunk 48]
```diff
@@ -462,6 +470,115 @@
...
+              }
+            },
+            {
+              "name": "Arm",
+              "layers": {
+                "0": "UTorso_Fleece_KelsRewardArm_Base.png"
+              }
+            }
+          ]
+        },
```
[the remark this answers is not in the document]
Comment (Hazado) [reply sources/avatar/clothing_fan02/Fan02.json:511]: Up to you...its reward clothing so if there is no way to get it. No one see it, similar to other clothing in the files that are unavailable for whatever reason

**Comment by Hoikas** [comment]
> Ok, so we probably need to drop the [MF]Reward] prefix in favor of matching... whatever the non-vault reward clothing does.

Although, now that I think about it a little more... Retaining the MF reward prefix for the Mysterium stuff might be OK in case we want to support vault granting, even if that's not really the intended mode of operation,

**Comment by Hazado** [comment]
> Ok, so we probably need to drop the `[MF]Reward]` prefix in favor of matching... whatever the non-vault reward clothing does.

Pretty sure non-vault reward clothing does the [MF]Reward as well. At least all the in-game clothing does

**Comment by Hoikas** [comment]
Some of it does, some of it doesn't; see `MTorso_Atrus`. The key is whether or not it needs to be vault grantable via Relto. For the Mysterium stuff, that's possibly desirable. For the Eder Naything one, probably not.

File: sources/avatar/clothing_fan02/Fan02.json (hunk 48 above)
[the remark this answers is not in the document]
Comment (Hazado) [reply sources/avatar/clothing_fan02/Fan02.json]: Ive changed this to [MF]Torso_RewKelsFleece and informed the age author of the change

File: sources/avatar/clothing_mysterium/mysterium.json (hunk 31 above)
Comment (Hazado) [reply sources/avatar/clothing_mysterium/mysterium.json]: Changed to MReward_SSMysterium2006a

**Comment by Hazado** [comment]
Naming convention for clothing
[MF]Reward_Name - clothing rewarded in Relto
[MF]BodyPart_Name - clothing rewarded in ages (Example: MTorso_CoolShirt)

**Review by Hoikas — APPROVED** [review]

**Pull request merged**